erpnext/schools/doctype/student/student.json

```diff
@@ -180,7 +180,7 @@
    "precision": "", 
    "print_hide": 0, 
    "print_hide_if_no_value": 0, 
-   "read_only": 1, 
+   "read_only": 0, 
```

Comment:
Hi Neil,

Yes, it's mandatory but the fact that this field is read-only, combined with the fact that the default value is Today makes it such that the value in this field will always be the creation date of the student record.

Regards,
Chude